src/postmate.js

```diff
@@ -32,6 +32,20 @@ function messageId() {
   return ++_messageId;
 }
 
+/**
+ * A unique child ID that is used to ensure responses are received from the correct child iframes
+ * @type {Number}
+ */
+let _childId = 0;
```

Comment:
I'm concerned Child Id's may not be unique with a  numeric iterator pattern like this.